src/main/java/com/iridium/iridiumskyblock/listeners/PlayerPortalListener.java

```diff
@@ -16,6 +16,19 @@ public class PlayerPortalListener implements Listener {
 
     @EventHandler(ignoreCancelled = true)
     public void onPlayerPortal(PlayerPortalEvent event) {
+
+        boolean setCancelled = true;
+        for(String world : IridiumSkyblock.getInstance().getConfiguration().whitelistedWorlds) {
+            if(event.getPlayer().getWorld().getName().equalsIgnoreCase(world)) {
+                setCancelled = false;
+                break;
+            }
+        }
+
+        if(setCancelled) {
+            return;
+        }
+
```

Comment:
This shouldnt be needed since we already do a check for if an island is present, which already does a World check